community/cypher/cypher-compiler-3.1/src/main/scala/org/neo4j/cypher/internal/compiler/v3_1/planner/logical/CardinalityCostModel.scala

```diff
@@ -38,16 +38,20 @@ object CardinalityCostModel extends CostModel {
 
   private def costPerRow(plan: LogicalPlan): CostPerRow = plan match {
 
-    case  _: AllNodesScan |
-         _: DirectedRelationshipByIdSeek |
-         _: UndirectedRelationshipByIdSeek |
+    case  _: NodeByLabelScan |
          _: ProjectEndpoints
     => FAST_STORE
 
-    case _: NodeByLabelScan => 1.6
+    case _: AllNodesScan => 1.2
 
     case _: Expand |
-         _: VarExpand  => 2.5
+         _: VarExpand  => 2.0
+
+    case _: NodeUniqueIndexSeek |
+         _: NodeIndexSeek |
+         _: NodeIndexContainsScan |
+         _: NodeIndexEndsWithScan |
+         _: NodeIndexScan => 3.0
```

Comment:
Given that this file has so many magic constants now, does it even make sense to have some of them named? Ie, `FAST_STORE`, `CPU_BOUND` etc. They seem as arbitrary as these new ones.